platform/o.n.core/src/org/netbeans/core/ui/ProductInformationPanel.java

```diff
@@ -102,6 +103,7 @@ public ProductInformationPanel() {
             final String updates = getUpdates();
             SwingUtilities.invokeLater(() -> {
                 description.setText(LBL_description(getProductVersionValue(), getJavaValue(), getVMValue(), getOperatingSystemValue(), getEncodingValue(), getSystemLocaleValue(), getUserDirValue(), Places.getCacheDirectory().getAbsolutePath(), updates, FONT_SIZE, getJavaRuntime()));
+                descriptionScrollPane.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_ALWAYS); // will need a scrollbar now
```

Comment:
It's not clear to me what `now` is referring to in the comment.

Reply:
the moment the description text is updated. Which is what the lambda does. So at the point this lambda runs.

Reply:
Thanks, I was confused by the title ".... scrollbar if the async plugin update..." but it's a little more like
"... scrollbar in case the async plugin update...".

I had thought it might mean now that there are updates available.
So the scrollbar is there, even when there are no updates available.